.github/workflows/track_benches.yml

```diff
@@ -0,0 +1,95 @@
+
+name: Track Benchmarks
+
+jobs:
+  track_benchmarks:
+    if: github.event.workflow_run.conclusion == 'success'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        backend: ["postgres", "sqlite", "mysql"]
+    env:
+      BENCHER_PROJECT: diesel
+      BENCHER_ADAPTER: rust_criterion
+      BENCHER_TESTBED: ubuntu-latest-${{ matrix.backend }}
+      PR_BENCHMARK_RESULTS: pr_${{ matrix.backend }}.txt
+      BASE_BENCHMARK_RESULTS: base_${{ matrix.backend }}.txt
+      GITHUB_EVENT: event_${{ matrix.backend }}.json
+    steps:
+      - name: Download Benchmark Results
+        uses: actions/github-script@v6
+        with:
+          script: |
+            async function downloadArtifact(artifactName) {
+              let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                run_id: context.payload.workflow_run.id,
+              });
+              let matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
+                return artifact.name == artifactName
+              })[0];
+              if (!matchArtifact) {
+                core.setFailed(`Failed to find artifact: ${artifactName}`);
+              }
+              let download = await github.rest.actions.downloadArtifact({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                artifact_id: matchArtifact.id,
+                archive_format: 'zip',
+              });
+              let fs = require('fs');
+              fs.writeFileSync(`${process.env.GITHUB_WORKSPACE}/${artifactName}.zip`, Buffer.from(download.data));
+            }
+            await downloadArtifact(process.env.PR_BENCHMARK_RESULTS);
+            await downloadArtifact(process.env.BASE_BENCHMARK_RESULTS);
+            await downloadArtifact(process.env.GITHUB_EVENT);
+      - name: Unzip Benchmark Results
+        run: |
+          unzip $PR_BENCHMARK_RESULTS.zip
+          unzip $BASE_BENCHMARK_RESULTS.zip
+          unzip $GITHUB_EVENT.zip
+      - name: Export GitHub Event Data
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let fs = require('fs');
+            let githubEvent = JSON.parse(fs.readFileSync("event.json", {encoding: 'utf8'}));
+            console.log(githubEvent);
+            core.exportVariable("PR_HEAD", `${githubEvent.pull_request.head.ref}-${githubEvent.pull_request.head.sha.slice(0, 8)}`);
+            core.exportVariable("PR_ID", `${githubEvent.pull_request.head.ref}/${process.env.BENCHER_TESTBED}/${process.env.BENCHER_ADAPTER}`);
+            core.exportVariable("PR_NUMBER", githubEvent.number);
+      - uses: bencherdev/bencher@main
+      - name: Track base Benchmarks
+        run: |
+          bencher run \
+          --if-branch '${{ env.PR_HEAD }}' \
+          --else-branch \
+          --token "${{ secrets.BENCHER_API_TOKEN }}" \
```

Comment:
As a note to myself: We should to set our own API token here before merging, right?

Reply:
Yes! Add `BENCHER_API_TOKEN` as a Repository secret. (ex: Repo -> Settings -> Secrets and variables -> Actions -> New repository secret) 
I have already started using the `diesel` project on Bencher, so I can just invite you to the project there, or you can make an entirely new project. If you choose the latter then we can do a slug-swap if you want to have the `/perf/diesel` URL going forward.

Reply:
If it's possible to remove you as owner from the existing project after I joined I would like to go that route.

Reply:
Yes, it is! I sent you an invite, so you should see that in you inbox. Once you accept, I can just remove myself from the existing project.